src/pages/promo-code/promo-code-page.js

```diff
@@ -135,6 +137,15 @@ const PromoCodePage = () => {
         >
           {config.titles.promoPageTitles.mainPageTitle}
         </Typography>
+        <Button
+          data-cy='add-certificate'
```

Comment:
it is not add-certificate, but it is better to remove this data-cy, you can get it by getByTitle or getByText.